.github/workflows/create-release.yml

```diff
@@ -0,0 +1,51 @@
+name: Create release
+
+on:
+  workflow_dispatch:
+    inputs:
+      mode:
+        type: choice
+        description: release type
+        required: true
+        default: 'minor'
+        options:
+          - patch
+          - minor
+          - major
+
+jobs:
+  run-release:
+    name: Demo
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch from origin repo
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+          token: ${{ secrets.TINKOFF_BOT_PAT }}
+
+      - name: Setup Node.js and Cache
+        uses: ./.github/actions/nodejs
+
+      - name: Run release
+        run: |
+          git config user.name "tinkoff-bot"
+          git config --global push.followTags true
+          npm run release -- --release-as ${{ github.event.inputs.mode }}`
+          npm run prettier
+          git add .
+          git commit --all --amend --no-edit --no-verify
+          NEW_VERSION=$(node -p "require('./package.json').version")
+          echo "new_version = $NEW_VERSION" >> $GITHUB_ENV
```

Comment:
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions
please don't use file overwriting with variables, it's bad practice

<img width="869" alt="image" src="https://user-images.githubusercontent.com/12021443/153849624-35d44171-0294-40a1-bc2d-e43d10b36f3f.png">

Reply:
thanks